health/main.sh

```diff
@@ -1,15 +1,9 @@
 #!/bin/bash
 
-LOG=/var/log/health
-
-touch $LOG
-
 if [ ! -z "$RUN_HEALTH_JOB" ]; then
-    python job.py >> $LOG &
+    python job.py &
 fi
 
-if [ ! -z "$RUN_HEALT_API" ]; then
-    gunicorn -w 4 -b 0.0.0.0:5000 main:app >> $LOG &
+if [ ! -z "$RUN_HEALTH_API" ]; then
+    gunicorn -w 4 -b 0.0.0.0:5000 main:app &
```

Comment:
The container which runs this script will die because no-one blocks execution of the script. I would suggest to run gunicorn in the end of this script in the foreground.